app/src/pages/SingleContent.spec.ts

```diff
@@ -0,0 +1,113 @@
+import "fake-indexeddb/auto";
+import { describe, it, expect, afterEach, beforeEach } from "vitest";
+import { mount, shallowMount } from "@vue/test-utils";
+import { setActivePinia, createPinia } from "pinia";
+import SingleContent from "./SingleContent.vue";
+import { mockPostDto, mockEnglishContentDto, mockCategoryContentDto } from "@/tests/mockData";
+import { db, type BaseDocumentDto } from "luminary-shared";
+import waitForExpect from "wait-for-expect";
+
+describe("SinglePost", () => {
+    beforeEach(() => {
+        db.docs.bulkPut([
+            mockPostDto,
+            mockEnglishContentDto,
+            mockCategoryContentDto,
+        ] as BaseDocumentDto[]);
+
+        setActivePinia(createPinia());
+    });
+
+    afterEach(() => {
+        db.docs.clear();
+    });
+
+    it("displays the loading when no content", async () => {
```

Comment:
Test description should read "displays the loading spinner when no content is available"